src/main/java/teammates/common/util/AppUrl.java

```diff
@@ -42,4 +42,8 @@ public AppUrl withQuestionNumber(String questionNumber) {
         return withParam(Const.ParamsNames.FEEDBACK_QUESTION_NUMBER, questionNumber);
     }
 
+    public AppUrl withEnableSessionEditDetails(String isLoadedInEditMode) {
```

Comment:
This can be a boolean too. Convert it to a string before passing to `withParam`.